config/initializers/mailer_previews.rb

```diff
@@ -1,10 +1,10 @@
-ActiveSupport.on_load(:action_controller, run_once: true) do
+Rails.application.config.after_initialize do
```

Comment:
This feels like a constant battle

Reply:
Actually, it might not be needed here now -- definitely if we upgrade, it's necessary, but possibly not yet. I'll revert until it's necessary.